sidebars.js

```diff
@@ -307,6 +307,14 @@ module.exports = {
         { type: 'doc', id: 'deployment_patterns/index', label: 'Index' }
       ]
     },
+    {
+      type: 'category',
+      label: 'Integrations',
+      items: [
+        'integrations/contributing_integration',
+        { type: 'doc', id: 'integrations/index', label: 'Index' }
+      ]
+    },
```

Comment:
I would put the documents on contributing integrations under the category "Contributing > Contributing specifics > Integrations" (that last category will need to be created.). I wold also include a link to the mentioned integrations template in that section, as per the "How to contribute a how-to guide" section.

You'll then want create another category at this level (or reuse this one) named "Integrations" to contain the actual integration documentation, and make sure it contains the relevant documents that currently reside in the Reference Architectures category.

Reply:
I'm a bit confused by this. Not sure I follow.

Reply:
@Rachel-Reverie I think I get it now. However, can we do the information architecture re-factor in a follow-up PR? I rather have this content out and then move things around as needed.

Reply:
That would be perfectly fine.